src/content/development/contribution-guide/_index.en.md

```diff
@@ -30,19 +30,41 @@ To be expanded
 
 ## Bugs, Tasks and Epics
 
-To be expanded
+In order to have more structure and clarity, we expand upon the standard GitHub issue and define 3 types of issues:
+Bugs, Tasks and Epics.
 
 ### Bugs
 
-To be expanded
+A bug is an issue which captures an error or fault in the project.
+If a bug meets the criteria of a blocker, it is considered a blocker bug.
+
+#### Blocker Criteria
+
+If an issue prevents a feature (either new or existing) from operating correctly and there's no sufficient workaround, it may be deemed as
+a blocker for a release such that the release cannot proceed until it is addressed.
 
 ### Tasks
 
-To be expanded
+A task defines a specific unit of work that can be stand-alone or part of an epic. A task should be relatively small and fit within
+the scope of a single sprint otherwise it should be broken down into smaller tasks or perhaps be defined as an epic.
 
 ### Epics
 
-To be expanded
+An epic is a collection of tasks which are required to accomplish a feature.
+
+#### Epic Guidelines
+
+* Epics should be issues in the [enhancements repository](https://github.com/submariner-io/enhancements/issues) and created
+using the [epic template](https://github.com/submariner-io/enhancements/issues/new?assignees=&labels=epic&template=epic.md).
+* Only include work directly related to the Submariner project.
```

Comment:
```suggestion
* Only include work that is a part of the Submariner project.
```

It seems to me "directly related" could mean lots of things we don't intend. For example, the submariner-addon for ACM or ACM's docs about Submariner seem directly related to Submariner but I think are not efforts we want to track here.